lib/rom/rails/railtie.rb

```diff
@@ -79,9 +79,12 @@ def setup
       end
 
       def infer_default_repository
-        return unless defined?(ActiveRecord)
-        spec = ActiveRecord::Configuration.call
-        [:sql, spec[:uri], spec[:options]]
+        if RepositoryConfiguration.inferrable?
+          spec = RepositoryConfiguration.infer
+          [:sql, spec[:uri], spec[:options]]
+        elsif ENV['DATABASE_URL']
+          [:sql, ENV['DATABASE_URL']]
```

Comment:
This should be part of the helper class too, it's yet another source of configuration in addition to DM and AR.